src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  flex-direction: row;
+  display: flex;
+  gap: 4px;
+
+  &__star {
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+
+    &:last-child {
+      background-image: url('../images/star.svg');
```

Comment:
Ensure that the image files `star-active.svg` and `star.svg` exist in the `../images/` directory. Incorrect paths or missing files will result in broken images in the UI.